src/locales/en-AU.json

```diff
@@ -242,35 +243,86 @@
   "mnoe_admin_panel.dashboard.finance.table.paid": "Paid",
   "mnoe_admin_panel.dashboard.finance.table.yes": "Yes",
   "mnoe_admin_panel.dashboard.finance.table.no": "No",
-  "mnoe_admin_panel.dashboard.staff.title": "Staff",
-  "mnoe_admin_panel.dashboard.staff.add_staff": "Add staff",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.title": "Add a new staff",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.cancel": "Cancel",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.save": "Save",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.placeholder.name": "First name",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.placeholder.surname": "Last name",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.placeholder.email": "Email",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.placeholder.role": "- Role -",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.toastr_success": "{{username}} has been successfully added.",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.toastr_error_email": "An error occurred while sending an email to {{email}}.",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.toastr_error": "An error occurred while adding {{user_name}}.",
-  "mnoe_admin_panel.dashboard.staff.add_staff.modal.toastr_success": "{{user_name}} has been successfully added.",
-  "mnoe_admin_panel.dashboard.staff.widget.list.title": "All Staff",
-  "mnoe_admin_panel.dashboard.staff.widget.list.table.first_name": "First name",
-  "mnoe_admin_panel.dashboard.staff.widget.list.table.last_name": "Last name",
-  "mnoe_admin_panel.dashboard.staff.widget.list.table.creation": "Creation",
-  "mnoe_admin_panel.dashboard.staff.widget.list.table.role": "Role",
-  "mnoe_admin_panel.dashboard.staff.widget.list.table.admin_role.all": "All",
-  "mnoe_admin_panel.dashboard.staff.widget.list.placeholder.search_name": "Search for name",
-  "mnoe_admin_panel.dashboard.staff.widget.list.placeholder.search_surname": "Search for surname",
-  "mnoe_admin_panel.dashboard.staff.widget.list.loading_staff": "Loading ... ",
-  "mnoe_admin_panel.dashboard.staff.widget.list.toastr_error": "An error occurred while saving the user.",
-  "mnoe_admin_panel.dashboard.staff.widget.list.toastr_success": "{{staff_name}} has been successfully removed.",
-  "mnoe_admin_panel.dashboard.staff.modal.remove_staff.cancel": "Cancel",
-  "mnoe_admin_panel.dashboard.staff.modal.remove_staff.delete": "Delete Team Member",
-  "mnoe_admin_panel.dashboard.staff.modal.remove_staff.proceed": "Delete {{staff_name}}?",
-  "mnoe_admin_panel.dashboard.staff.modal.remove_staff.perform": "Are you sure you want to delete this team member?",
-  "mnoe_admin_panel.dashboard.staff.modal.remove_staff.toastr_error": "An error occured while deleting the user.",
+  "mnoe_admin_panel.dashboard.staffs.title": "Staffs",
+  "mnoe_admin_panel.dashboard.staffs.add_staff": "Add staff",
+  "mnoe_admin_panel.dashboard.staffs.add_staff.modal.title": "Add a new staff",
```

Comment:
"Add a new staff member"
Also this is not consistent with "Are you sure you want to delete this team member?"
Title should be updated too